quint/src/generated/Quint.g4

```diff
@@ -150,14 +150,17 @@ unitOrExpr :    unit | expr;
 // 1. x => e
 // 2. (x) => e
 // 3. (x, y, z) => e
-lambda:         identOrHole '=>' expr
-        |       '(' identOrHole (',' identOrHole)* ')' '=>' expr
+lambda:         parameter '=>' expr
+        |       '(' parameter (',' parameter)* ')' '=>' expr
         ;
 
+
 // an identifier or a hole '_'
 identOrHole :   '_' | IDENTIFIER
         ;
 
+parameter: identOrHole;
```

Comment:
This is not completely necessary, but makes the code in `ToIRListener` more consistent, and I think it makes some sense to differentiate parameters from simple identifiers at the grammar level: parameters will later become a `QuintLambdaParameter`, while `identOrHole` is just used as a string.

Reply:
Good call I think this makes great sense!